### PR TITLE
alex/more swiftlint fixes

### PR DESCRIPTION
**Pull request by alexpersian**
* Addresses the final lint warnings

### CODE REVIEW

File: .swiftlint.yml [hunk 1]
```diff
@@ -115,3 +115,7 @@ nesting:
 
 identifier_name:
   allowed_symbols: _
+  min_length: 2
+
```
Comment (alexpersian) [inline .swiftlint.yml:119]: These two changes are to address our use of `Qu` and `id` naming conventions.

**Comment by IntrepidBuildServer** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          4 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/IntrepidPursuits/swift-wisdom/blob/bf76194b076a1a933c8bedb3068bf7b9426f3644/Array+UtilitiesTests.swift#L100">Array+UtilitiesTests.swift#L100</a> - <strong><a href="https://github.com/IntrepidPursuits/swift-wisdom/blob/bf76194b076a1a933c8bedb3068bf7b9426f3644/SwiftWisdomTests/StandardLibrary/Array/Array+UtilitiesTests.swift#L100">SwiftWisdomTests/StandardLibrary/Array/Array+UtilitiesTests.swift#L100</a></strong>: ‘subscript(ip_safe:)’ is deprecated: Use <code>ip_safely</code> instead.  <br /><code>
        XCTAssert(testArray[ip_safe: 3] == 4)
</code></td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/IntrepidPursuits/swift-wisdom/blob/bf76194b076a1a933c8bedb3068bf7b9426f3644/Array+UtilitiesTests.swift#L101">Array+UtilitiesTests.swift#L101</a> - <strong><a href="https://github.com/IntrepidPursuits/swift-wisdom/blob/bf76194b076a1a933c8bedb3068bf7b9426f3644/SwiftWisdomTests/StandardLibrary/Array/Array+UtilitiesTests.swift#L101">SwiftWisdomTests/StandardLibrary/Array/Array+UtilitiesTests.swift#L101</a></strong>: ‘subscript(ip_safe:)’ is deprecated: Use <code>ip_safely</code> instead.  <br /><code>
        XCTAssert(testArray[ip_safe: 8] == 9)
</code></td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/IntrepidPursuits/swift-wisdom/blob/bf76194b076a1a933c8bedb3068bf7b9426f3644/Array+UtilitiesTests.swift#L102">Array+UtilitiesTests.swift#L102</a> - <strong><a href="https://github.com/IntrepidPursuits/swift-wisdom/blob/bf76194b076a1a933c8bedb3068bf7b9426f3644/SwiftWisdomTests/StandardLibrary/Array/Array+UtilitiesTests.swift#L102">SwiftWisdomTests/StandardLibrary/Array/Array+UtilitiesTests.swift#L102</a></strong>: ‘subscript(ip_safe:)’ is deprecated: Use <code>ip_safely</code> instead.  <br /><code>
        XCTAssertNil(testArray[ip_safe: -1])
</code></td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/IntrepidPursuits/swift-wisdom/blob/bf76194b076a1a933c8bedb3068bf7b9426f3644/Array+UtilitiesTests.swift#L103">Array+UtilitiesTests.swift#L103</a> - <strong><a href="https://github.com/IntrepidPursuits/swift-wisdom/blob/bf76194b076a1a933c8bedb3068bf7b9426f3644/SwiftWisdomTests/StandardLibrary/Array/Array+UtilitiesTests.swift#L103">SwiftWisdomTests/StandardLibrary/Array/Array+UtilitiesTests.swift#L103</a></strong>: ‘subscript(ip_safe:)’ is deprecated: Use <code>ip_safely</code> instead.  <br /><code>
        XCTAssertNil(testArray[ip_safe: 9])
</code></td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">Executed 142 tests, with 0 failures (0 unexpected) in 0.301 (0.399) seconds</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: SwiftWisdom/Core/UIKit/UIView+GetConstraints.swift [hunk 2]
```diff
@@ -3,17 +3,12 @@
 ////  Created by Paul Rolfe on 7/31/15.
 ////  Copyright (c) 2015 Paul Rolfe. All rights reserved.
 ////
```
Comment (alexpersian) [inline SwiftWisdom/Core/UIKit/UIView+GetConstraints.swift:5]: This file was just white space and alignment fixes. The comment removed was a duplicate.

**Review by ButkiewiczP — APPROVED** [review]

**Comment by ghost** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          4 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/IntrepidPursuits/swift-wisdom/blob/7f3c5e7a979f5d46d1fcdd9754490f6e31c95c89/Array+UtilitiesTests.swift#L100">Array+UtilitiesTests.swift#L100</a> - <strong><a href="https://github.com/IntrepidPursuits/swift-wisdom/blob/7f3c5e7a979f5d46d1fcdd9754490f6e31c95c89/SwiftWisdomTests/StandardLibrary/Array/Array+UtilitiesTests.swift#L100">SwiftWisdomTests/StandardLibrary/Array/Array+UtilitiesTests.swift#L100</a></strong>: ‘subscript(ip_safe:)’ is deprecated: Use <code>ip_safely</code> instead.  <br /><code>
        XCTAssert(testArray[ip_safe: 3] == 4)
</code></td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/IntrepidPursuits/swift-wisdom/blob/7f3c5e7a979f5d46d1fcdd9754490f6e31c95c89/Array+UtilitiesTests.swift#L101">Array+UtilitiesTests.swift#L101</a> - <strong><a href="https://github.com/IntrepidPursuits/swift-wisdom/blob/7f3c5e7a979f5d46d1fcdd9754490f6e31c95c89/SwiftWisdomTests/StandardLibrary/Array/Array+UtilitiesTests.swift#L101">SwiftWisdomTests/StandardLibrary/Array/Array+UtilitiesTests.swift#L101</a></strong>: ‘subscript(ip_safe:)’ is deprecated: Use <code>ip_safely</code> instead.  <br /><code>
        XCTAssert(testArray[ip_safe: 8] == 9)
</code></td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/IntrepidPursuits/swift-wisdom/blob/7f3c5e7a979f5d46d1fcdd9754490f6e31c95c89/Array+UtilitiesTests.swift#L102">Array+UtilitiesTests.swift#L102</a> - <strong><a href="https://github.com/IntrepidPursuits/swift-wisdom/blob/7f3c5e7a979f5d46d1fcdd9754490f6e31c95c89/SwiftWisdomTests/StandardLibrary/Array/Array+UtilitiesTests.swift#L102">SwiftWisdomTests/StandardLibrary/Array/Array+UtilitiesTests.swift#L102</a></strong>: ‘subscript(ip_safe:)’ is deprecated: Use <code>ip_safely</code> instead.  <br /><code>
        XCTAssertNil(testArray[ip_safe: -1])
</code></td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/IntrepidPursuits/swift-wisdom/blob/7f3c5e7a979f5d46d1fcdd9754490f6e31c95c89/Array+UtilitiesTests.swift#L103">Array+UtilitiesTests.swift#L103</a> - <strong><a href="https://github.com/IntrepidPursuits/swift-wisdom/blob/7f3c5e7a979f5d46d1fcdd9754490f6e31c95c89/SwiftWisdomTests/StandardLibrary/Array/Array+UtilitiesTests.swift#L103">SwiftWisdomTests/StandardLibrary/Array/Array+UtilitiesTests.swift#L103</a></strong>: ‘subscript(ip_safe:)’ is deprecated: Use <code>ip_safely</code> instead.  <br /><code>
        XCTAssertNil(testArray[ip_safe: 9])
</code></td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">Executed 142 tests, with 0 failures (0 unexpected) in 0.247 (0.322) seconds</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Pull request merged**